hack/util

```diff
@@ -1,44 +1,34 @@
 #!/usr/bin/env sh
 
-: ${PREFER_BUILDCTL=}
-: ${PREFER_LEGACY=}
+: ${CONTINUOUS_INTEGRATION=}
+: ${CI=}
+: ${TRAVIS_BRANCH=}
+: ${GITHUB_ACTIONS=}
 : ${TRAVIS_EVENT_TYPE=}
 : ${TRAVIS_BRANCH=}
+: ${CACHE_REF=}
 
-newerEqualThan() { # $1=minimum wanted version $2=actual-version
-  [ "$1" = "$(printf "$1\n$2" | sort -V | head -n 1)" ]
-}
-
-buildmode="legacy"
```

Comment:
this breaks the scripts that have not been updated. `update-vendor`, `shfmt` etc